tests/unit/Endpoints/PaymentsTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Alma\API\Tests\Unit;
+
+use Mockery;
```

Comment:
Mockery is a relatively modern mocker.
I've chosen this dependency because it's used by laravel, so it's well maintained and easy to use.

Reply:
:+1:

Reply:
@PaulRouss3l we need the API client to work with older versions of PHP (5.5/5.6).
Won't that new dependency break this compatibility?

Reply:
@olance It's only a dev dependency so it shouldn't be impact-full but we use mockery-1.3 which is compatible with php >=5.6.0, do we really need php5.5 ?

Reply:
Ah right, sorry I missed the dev-only part :)

All good!